db/mysql/schema.sql

```diff
@@ -306,6 +355,12 @@ CREATE TABLE `usr_user_role`
   PRIMARY KEY (`user_id`, `role_id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `usr_extuser` (
```

Comment:
Well, but then there is no point in having external user table, right? Because internal user ID is equal to CAS user id (it is provided by CAS if SSO is enabled). So external user table is not needed and may be removed, am I right?